src/pages/server-side-table-property-filter/index.jsx

```diff
@@ -94,6 +94,7 @@ function ServerSidePropertyFilterTable({ columnDefinitions, saveWidths, loadHelp
       wrapLines={preferences.wrapLines}
       stripedRows={preferences.stripedRows}
       contentDensity={preferences.contentDensity}
+      stickyColumns={preferences.stickyColumns}
```

Comment:
Should this be `{...preferences}` so we won't have to make that many changes every time?

Reply:
Sounds like a meaningful cleanup. Added to my todo.